indigo/distribution.yaml

```diff
@@ -10806,6 +10806,21 @@ repositories:
       url: https://github.com/rst-tu-dortmund/teb_local_planner.git
       version: master
     status: developed
+  teleop-twist-joy-xbot:
+    doc:
+      type: git
+      url: https://github.com/giupal/teleop_twist_joy_xbot-devel.git
+      version: 0.1.2
```

Comment:
This still needs to be a branch name which it isn't. Maybe `master`?

Same for the `source` entry.